internal/signalfx-agent/pkg/core/propfilters/filter_test.go

```diff
@@ -245,8 +245,8 @@ func TestFilters(t *testing.T) {
 		filteredDimension := f.FilterDimension(dim)
 		nodeFilteredDimension := f.FilterDimension(dimNode)
 		expectedProperties := map[string]string{"replicaSet": "abc", "service_uid": "123"}
-		assert.Equal(t, filteredDimension.Properties, expectedProperties)
-		assert.Equal(t, nodeFilteredDimension.Properties, properties)
+		assert.Equal(t, expectedProperties, filteredDimension.Properties)
+		assert.Equal(t, properties, nodeFilteredDimension.Properties)
```

Comment:
Shooting from the hip and not from your change but I am questioning if the test is checking the right thing: `properties` and `nodeProperties` are maps defined with the same keys and values. Isn't the case the if you filter both with the same filter they will have the same content? Anyway, not related to this change.

Reply:
I think line [228](https://github.com/signalfx/splunk-otel-collector/blob/db32e9b91f8b065fcb4b65f880420295f9d307d7/internal/signalfx-agent/pkg/core/propfilters/filter_test.go#L228) calling [New()](https://github.com/signalfx/splunk-otel-collector/blob/b2b8efe795c73c24431e6d21f2d679a6c40c8d1b/internal/signalfx-agent/pkg/core/propfilters/dimfilter.go#L32) with two different arguments for propertyNames and dimensionValues shows some insight into what is being tested here.